README.md

```diff
@@ -60,6 +60,10 @@ insert into db1.table1 values (1, 'a');
 select * from db1.table1;
 ```
 
+Apache Polaris supports the following optional build options:
+- `-PeclipseLink=true` – Enables the EclipseLink extension.
+- `-PeclipseLinkDeps=[groupId]:[artifactId]:[version],...` – Specifies one or more additional dependencies for EclipseLink (e.g., JDBC drivers), separated by commas.
```

Comment:
Not sure if it is a good idea to add examples for Postgres or mySQL. Thoughts? cc @aihuaxu. Not a blocker though.